astester.imports.js

```diff
@@ -172,6 +172,9 @@ export function local(memory) {
         ledger.set(newAddress._value, newAddressLedger);
         return newString(newAddress._value);
       },
+      assembly_script_get_time(){
+        return BigInt(Date.now());  
```

Comment:
You should force the conversion to an u64 value yourself.

```suggestion
        return BigInt.asUintN(64, Date.now());  
```